tools/ansible/tasks/wsu/main.yaml

```diff
@@ -154,6 +161,17 @@
       shell: "oc label node {{ node_name.stdout_lines[0] }} node-role.kubernetes.io/worker="
       when: node_name.stdout_lines | length == 1 and checklabels.rc == 1
 
+    - name: Check if the hybrid overlay is running
+      win_shell: "Get-Process -Name \"hybrid-overlay\""
+      register: get_process
+      failed_when:
+        - 'get_process.stderr != ""'
+        - '"Cannot find a process with the name" not in get_process.stderr'
+
+    - name: Stop the hybrid overlay if it is running
+      win_shell: "Stop-Process -Name \"hybrid-overlay\""
```

Comment:
Don't you need to check if the hybrid-overlay actually stopped and no errors were returned?

Reply:
If the command returns a non-zero return code the module will fail:
https://github.com/ansible/ansible/blob/4676e5d60e568f20ebf2791ea7dc39ea61b15455/lib/ansible/modules/windows/win_shell.ps1#L134-L136